Apps/WebAPI/sdLitica.WebAPI/Controllers/v1/AnalyticsController.cs

```diff
@@ -116,10 +120,53 @@ public IActionResult GetUserOperation([FromRoute] string userOperationId)
         [Route("operations/{userOperationId}/result")]
         public IActionResult GetUserOperationResult([FromRoute] string userOperationId)
         {
-            UserAnalyticsOperation userAnalyticsOperation = _analyticsService.GetUserAnalyticsOperation(userOperationId);
-            if (userAnalyticsOperation == null || userAnalyticsOperation.Status != OperationStatus.Complete) return NotFound();
+            UserAnalyticsOperation userAnalyticsOperation =
+                _analyticsService.GetUserAnalyticsOperation(userOperationId);
+            if (userAnalyticsOperation == null || userAnalyticsOperation.Status != OperationStatus.Complete)
+                return NotFound();
 
             return Ok(); // todo: return result of operation
         }
+
+        /// <summary>
+        /// Get list of user's analytical operations by timeseries id
+        /// </summary>
+        /// <param name="seriesId">id of timeseries</param>
+        [HttpGet]
+        [Route("operations/bySeriesId/{seriesId}")]
+        public IActionResult GetUserOperationsByTimeSeries([FromRoute] string seriesId)
+        {
+            List<UserAnalyticsOperation> operations =
+                _analyticsService.GetUserAnalyticsOperationsBySeriesId(seriesId);
+            if (operations == null) return NotFound();
+
+            List<UserAnalyticsOperationModel> models = operations.Select(operation => new UserAnalyticsOperationModel
+                {
+                    Id = operation.Id.ToString(),
+                    OperationName = operation.OperationName,
+                    Status = operation.Status.ToString(),
+                    TimeSeriesId = operation.TimeSeriesId
+                })
+                .ToList();
+
+            return Ok(models);
+        }
+
+        /// <summary>
+        /// Get list of results of user's analytical operations by timeseries id
+        /// </summary>
+        /// <param name="seriesId">id of timeseries</param>
+        /// <returns></returns>
+        [HttpGet]
+        [Route("operations/{userOperationId}/result")]
```

Comment:
route duplicates the one of GetUserOperationResult()

Reply:
fixed